### DISCUSSION TITLE
Translation issues: how often are localization files updated?

### DISCUSSION THREAD

**Discussion by Ondrej-Janca** [gephi/gephi#2754] [Q&A] [answered]
Hey,

I am native Czech speaker and today I changed language to Czech for the first time and immediately noticed inconsistencies and mistranslations in Gephi Czech localization.

I found the Weblate translation project and I will help there. I just wanted to ask, when are the translation updates incorporated. Is it with every new version launch?

**Comment by mbastian** [comment, marked as answer]
Thanks Ondrej for volunteering to contribute. We haven't had anyone touch the Czech translation in a while and I'm sure it can be improved.

Changes from Weblate are automatically pushed as PR in this repository so we regularly integrate them. We would certainly do so ahead of a release.

**Reply by Ondrej-Janca** [reply]
Alright,
I will go through the translation in a about month's time (I have a publication deadline coming).
There is nothing that would prevent the users from using the app, as far as I saw.
